scripts/sync_data.py

```diff
@@ -59,6 +65,27 @@ def download_contrarianmba_airtable_records():
         f.write("\n")
 
 
+def download_product_images():
+    with open(CONTRARIANMBA_JSON_FILENAME, "r") as f:
+        data = json.loads(f.read())
+        books = [book for book in data["lookups"]["book_id"].values()]
+
+    for book in books:
+        image_url = build_amazon_image_url(book["amazonProductID"])
+        image_path = f"{PUBLIC_FOLDER}/images/products/{book['amazonProductID']}.jpg"
+
+        if os.path.exists(image_path):
```

Comment:
Use pathlib instead of `os` since we’re already using pathlib

https://docs.python.org/3/library/pathlib.html#pathlib.Path.exists